src/components/Typography/Typography.tsx

```diff
@@ -32,6 +32,7 @@ const DynamicText = ({
     monospace,
     copyable,
     children,
+    iconSize = 22,
```

Comment:
default Icon size for CopyButton is 1em, which means it takes the max height of the parent element. Why do we hardcode here `22`?

Reply:
Did'nt see the 1em, good catch bro